### DISCUSSION TITLE
`whereIn()->exists()` problem

### DISCUSSION THREAD

**Discussion by foremtehan** [laravel/framework#51412] [General]
![image](https://github.com/laravel/framework/assets/53290883/41e198b2-dbb8-4e9e-9d11-b9a473a5b7a5)

Is this correct behavior? The second item does not exist

**Comment by ghost** [comment]
Hi. This is intended behavior. In means 1 or 100...;

**Comment by faissaloux** [comment]
If you have an `Author` with id in the array you provided, yes correct.
